### PR TITLE
[#5426] Fix force flag usage in rsStructFileBundle (4-2-stable)

### PR DESCRIPTION
**Pull request by alanking**
cherry-picked from changes for #5629 

Haven't run this yet but I don't imagine it will be much different from master, so I'm making a PR

### CODE REVIEW

**Comment by alanking** [comment]
CI tests running

**Comment by alanking** [comment]
CI tests passed except for 2 which appear to be due to disk I/O thrashing. Going to run these again to make sure.

**Comment by alanking** [comment]
Tests pass

File: scripts/irods/test/test_ibun.py [hunk 1]
```diff
@@ -11,17 +11,17 @@
...
 class Test_Ibun(resource_suite.ResourceBase, unittest.TestCase):
-
     def setUp(self):
         super(Test_Ibun, self).setUp()
         self.known_file_name = 'known_file'
 
     def tearDown(self):
         super(Test_Ibun, self).tearDown()
 
+    @unittest.skip('Generation of large file causes I/O thrashing... skip for now')
```
[the remark this answers is not in the document]
Comment (alanking) [reply scripts/irods/test/test_ibun.py:24]: I didn't remove this for master, but I suppose it's fine to not skip these. Might cause problems for those who exhibit the I/O thrashing (i.e. me :)). I'll do it, just have to make another PR for master before we close issue. Thoughts?

[the remark this answers is not in the document]
Comment (trel) [reply scripts/irods/test/test_ibun.py:24]: fine skipping them if that's deemed good.    we're optimizing for the developer (you) at this time.  :)

[the remark this answers is not in the document]
Comment (alanking) [reply scripts/irods/test/test_ibun.py:24]: I ran the tests in CI and it took about 30 minutes with lots of thrashing on my machine, so I still vote that we continue to skip these for now. The plus side is that they passed.

**Comment by alanking** [comment]
#'d

**Pull request merged**